Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -28,7 +28,12 @@ extension KeyboardShortcuts {
 	public final class RecorderCocoa: NSSearchField, NSSearchFieldDelegate {
 		private let minimumWidth: Double = 130
 		private var eventMonitor: LocalEventMonitor?
-		private let shortcutName: Name
+		/// shortcutName should be updated by NSViewRepresentable.updateView from Recorder and same as stringValue
```

Comment:
The comment should be a doc comment about what it does for the user, not mention internal implementation details.